deployer/src/lib.rs

```diff
@@ -143,6 +143,10 @@ pub struct StartOptions {
     /// Override the core agent's reconcile idle period
     #[structopt(long)]
     pub reconcile_idle_period: Option<humantime::Duration>,
+
+    /// Amount of time to wait for all containers to start.
+    #[structopt(short = "w", long)]
+    pub start_wait_timeout: Option<humantime::Duration>,
```

Comment:
we're already on the options for the `start`, what about just `wait_timeout` ?

Reply:
:+1: Done